components/services/bat_ads/public/cpp/ads_client_mojo_bridge.cc

```diff
@@ -115,6 +115,40 @@ void AdsClientMojoBridge::LoadResourceForId(
   std::move(callback).Run(ads_client_->LoadResourceForId(id));
 }
 
+void AdsClientMojoBridge::ClearScheduledCaptcha() {
+  ads_client_->ClearScheduledCaptcha();
+}
+
+// static
+void AdsClientMojoBridge::OnGetScheduledCaptcha(
+    CallbackHolder<GetScheduledCaptchaCallback>* holder,
+    const std::string& captcha_id) {
+  DCHECK(holder);
+
+  if (holder->is_valid()) {
+    std::move(holder->get()).Run(captcha_id);
+  }
+
+  delete holder;
+}
+
+void AdsClientMojoBridge::GetScheduledCaptcha(
+    const std::string& payment_id,
+    GetScheduledCaptchaCallback callback) {
+  // this gets deleted in OnGetScheduledCaptcha
+  auto* holder = new CallbackHolder<GetScheduledCaptchaCallback>(
```

Comment:
as above this stuff isn't needed when we're not translating between std::bind and base::Bind

Reply:
Fixed.